src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenNullCoalescingAssignmentTests.cs

```diff
@@ -145,67 +147,97 @@ public void FieldLvalue()
 public class C
 {
     int? f1 = null;
+    object f2 = null;
 
     public static void Main()
     {
         var c = new C();
         Console.WriteLine(c.f1 ??= GetInt());
+        Console.WriteLine(c.f2 ??= GetObject());
```

Comment:
> Console.WriteLine(c.f2 ??= GetObject()); [](start = 7, length = 41)

Consider extracting both `WriteLine` statements to separate methods for clarity in `VerifyIL`.